cli/tests/bundle/sourcemap_tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
```

Comment:
These should be mutually exclusive, is that intentional?

Reply:
Yes, for the below test:
```rs
 itest!(bundle_errors_on_two_source_map_options {
    args: "bundle jsx_import_from_ts.ts --config bundle/sourcemap_tsconfig.json",
    exit_code: 1,
    output_str: Some("[WILDCARD]\nerror: TS5053 [ERROR]: Option 'sourceMap' cannot be specified with option 'inlineSourceMap'.\n"),
  });
```